js_modules/dagster-ui/packages/ui-core/src/instance/BackfillPartitionsRequestedDialog.tsx

```diff
@@ -19,15 +23,16 @@ export const BackfillPartitionsRequestedDialog = ({backfill, onClose}: Props) =>
       }
       onClose={onClose}
     >
-      <DialogBody>
-        {backfill && backfill.partitionNames ? (
-          <Box flex={{direction: 'column', gap: 8}} style={{maxHeight: '80vh', overflowY: 'auto'}}>
-            {backfill.partitionNames.map((partitionName) => (
-              <div key={partitionName}>{partitionName}</div>
-            ))}
-          </Box>
-        ) : null}
-      </DialogBody>
+      <div style={{height: '340px', overflow: 'hidden'}}>
+        <VirtualizedItemListForDialog
+          items={[...(backfill?.partitionNames || [])].sort((a, b) => COLLATOR.compare(a, b))}
```

Comment:
This looks good to me! I commented down below about Dialog but it might be nice to throw an "if open, items={...}, else items={[]}" here. I'm not sure whether this requested dialog exists in any list views, but the partitionNames list can be 100k+ items so it'd be nice not to do the sorting until the list is onscreen.

Reply:
Ah ok, I'll at least bury the sort under another component.